hardware/Tado.cpp

```diff
@@ -22,26 +22,28 @@
 #include "../json/json.h"
 #include "../webserver/Base64.h"
 #include "Tado.h"
-#include <regex>
-#include <initializer_list>
 
-#define round(a) ( int ) ( a + .5 )
+using namespace std;
```

Comment:
Please do not use the 'using namespace std', but write out the complete name like std::string, also in the header

Reply:
Reverted change I made. Both header and cpp file now use explicit std namespace reference.